src/components/form/CheckboxField.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { action } from '@storybook/addon-actions';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { Form } from 'react-final-form';
+import { CheckboxField as CB } from './CheckboxField';
+
+export default { title: 'Components/CheckboxField' };
+
+export const CheckboxField = () => (
```

Comment:
Could you scope all of these under `Forms`? Also if there's only one story I leave out the name in the title to prevent unnecessary nesting.

Actually maybe it would make sense to move Field there as well.

```suggestion
export default { title: 'Components/Forms/Fields' };

export const Checkbox = () => (
```

That way the UI shows
```
Components/Forms/Fields/Checkbox
                        Date
                        Email
                        ...
```

`FieldGroup` and `SubmitButton` shouldn't be under `Fields` though, just `Forms`